test/utils/pickers/describeValue/testPickerActionBar.tsx

```diff
@@ -222,13 +223,12 @@ export const testPickerActionBar: DescribeValueTestSuite<any, 'picker'> = (
     });
 
     describe('today action', () => {
-      // TODO: temporary for vitest. Can move to `vi.useFakeTimers`
-      let timer: SinonFakeTimers | null = null;
       beforeEach(() => {
-        timer = useFakeTimers({ now: new Date(2018, 0, 1), toFake: ['Date'] });
+        vi.setSystemTime(new Date(2020, 0, 1));
```

Comment:
What is the reason behind the year change? 🤔

Reply:
probably copy/paste mistake